typescript/cli/src/utils/keys.ts

```diff
@@ -1,18 +1,109 @@
-import { ethers } from 'ethers';
+import { input } from '@inquirer/prompts';
+import { ethers, providers } from 'ethers';
 
-import { ensure0x } from '@hyperlane-xyz/utils';
+import { Address, ensure0x } from '@hyperlane-xyz/utils';
+
+import { ContextSettings, KeyConfig } from '../context.js';
+
+import { impersonateAccount } from './fork.js';
+
+const ETHEREUM_ADDRESS_LENGTH = 42;
```

Comment:
Please de-dupe with const in fork and use address utilities form `utils` package instead